cmd/create.go

```diff
@@ -475,6 +475,14 @@ func (c createConfig) prompt(client *fn.Client) (createConfig, error) {
 		return c, err
 	}
 
+	executable, err := os.Executable()
+	if err != nil {
+		fmt.Printf("Unable to get the executable path: %v", err)
+		executable = "func"
+	}
+	fmt.Printf("The command below is a shorthand method to create the same function!\n")
+	fmt.Printf("    %v -l %v -t %v %v\n", executable, c.Runtime, c.Template, c.Path)
```

Comment:
Looks good!  Of course, this is a bit more complicated to do correctly, so we might want to add a little bit more logic here to handle other flags, and omit those that were not changed from their defaults.

This is opening the way to a deep rabbit hole, but perhaps we could solve it like this:  Append the flags to the printed command IFF the user provided a value that is different from the flag static default. 

For example, passing `cmd` through `newCreateConfig` you could add a print [right after prompting](https://github.com/knative-sandbox/kn-plugin-func/pull/816/files#diff-e52f1f4fa0c533e3287cbe2ef0436582b5ca471a3fddcdf1f49fecd1ec0c7d13R269-R271) that builds a short command based on the current state of the system post-prompting:
```go
  // IN confirm mode.  If also in an interactive terminal, run prompts.
  if interactiveTerminal() {
    cfg.prompt(client)
    fmt.Println("Command:")
    fmt.Println(singleCommand(cmd, args, cfg))
  }
```
where `singleCommand` only prints flags which were meaningfully altered (warning, minimally tested!)
```go
// singleCommand prints the single command that could be used by the current
// user to minimally recreate the state of the current execution state.
func singleCommand(cmd *cobra.Command, args []string, cfg createConfig) string {
  var b strings.Builder
  b.WriteString(os.Args[0])           // process executable
  b.WriteString(" -l " + cfg.Runtime) // language runtime is required
  if cmd.Flags().Lookup("template").Changed {
    b.WriteString(" -t " + cfg.Template)
  }
  if cmd.Flags().Lookup("repository").Changed {
    b.WriteString("-r " + cfg.Repository)
  }
  if cmd.Flags().Lookup("verbose").Changed {
    b.WriteString(fmt.Sprintf(" -v %v", cfg.Verbose))
  }
  if len(args) > 0 {
    b.WriteString(" " + os.Args[0]) // optional trailing <path> argument
  }
  return b.String()
}
```

Reply:
Thanks for sharing the code segment.
I have changed:
```
if len(args) > 0 {
    b.WriteString(" " + os.Args[0]) // optional trailing <path> argument
  }
```
  to
```
if len(args) > 0 {
    b.WriteString(" " + cfg.Path) // optional trailing <path> argument
}
```
Hope this is fine.